.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -4,40 +4,72 @@ about: Create a report to help us improve
 labels: Bug, Pending
 assignees: ''
+
 ---
 
-### [GitHub issues](https://github.com/microsoft/botframework-webchat/issues) should be used for bugs and feature requests. See the Support section to get support related to Bot Framework and Web Chat.
+<!--
+Thanks for posting the issue to us. Before posting, please remove any personal-identifiable information, secret, token, or conversation ID.
+
+To expedite the fix, please follow this template and fill out as concise as possible.
+-->
+
+## Reproduce steps
+
+1. Using style options, set background color to red
+1. Type `herocard` to render a hero card
+
+### Expectations
+
+It should display the hero card with white background.
+
+But instead, it displayed the hero card with red background.
+
+<!-- If it did not render anything, please open Developer Tools in your browser and paste the console log here -->
+
+<!-- Please attach any screenshots with annotations, if any -->
+
+## Versions
 
-## Screenshots
+I tested on these versions of browsers:
 
-<!-- If applicable, add screenshots to help explain your problem. -->
-<!-- Be sure to remove or obscure personally identifiable information from your code and screenshots -->
+- [ ] Chrome: `latest`
+- [ ] Edge: `latest`
+- [ ] Firefox: `latest`
+- [ ] Internet Explorer: `11`
 
-## Version
+<!-- Please indicate if you host Web Chat under non-browser environment, e.g. Microsoft Teams, React Native, iOS WebView -->
 
-<!-- What version of Web Chat are you using? Are you using the CDN? NPM package? Or embedding Web Chat to your site via `<iframe>`? -->
-<!-- The fastest way to find your Web Chat version is by checking the meta tag on your deployed site. -->
+I tested on these versions and variants of Web Chat:
 
-## Describe the bug
+- [ ] Thru CDN, hosted by `https://cdn.botframework.com/`
+- [ ] Thru NPM, React version of `16.x.x`
+- [ ] Thru IFRAME, hosted by `https://webchat.botframework.com/embed/`
 
-<!-- Give a clear and concise description of what the bug is. -->
-<!-- Please be sure to add screenshots of the console errors in your browser, if there are any -->
+```
+botframework-webchat:bundle:variant=
+botframework-webchat:bundle:version=
+botframework-webchat:core:version=
+botframework-webchat:ui:version=
+```
 
-## To Reproduce
+<!-- Please open Developer Tools and run the following script and paste the content:
 
-Steps to reproduce the behavior:
+[].map.call(document.querySelectorAll('meta[name^="botframework"]'), function (m) { return m.name + '=' + m.content; }).join('\n')
+-->
 
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+## Steps to fix
 
-## Expected behavior
+I think the background color of hero card should default to white color, and settable thru style options.
 
-<!-- Give a clear and concise description of what you expect to happen when following the reproduction steps above. -->
+## Additional contexts
 
-## Additional context
+I am rendering an Adaptive Card and here is the content of the card.
```

Comment:
I tend to think that having text like this will just confuse the user. This should be laid out in a comment, like:
Please add additional context, such as: "I am rendering an Adaptive Card and here is the copy-pasted `json`:"